### DISCUSSION TITLE
New subdomain requires TXT verification, despite domain being pointed towards Vercel - every other subdomain works without setup

### DISCUSSION THREAD

**Discussion by carstenhag** [vercel/community#6297] [Account Issue] [answered]
### Question

For some reason I can't use my domain (subdomain) at this project: https://vercel.com/carsten-hagemanns-projects/darfichkiffen-chagemann-de

It asks me to verify it with a TXT entry, but that doesn't make sense to me. Also, creating a TXT entry doesn't even work.

I checked some other similar discussions but I couldn't tell whether it was the same thing or not. Could it be related to the team slug? In any case, it seems like this is an issue on the Vercel migration, if I don't get informed about any move and then it breaks future setups :D 

![chrome_EtsTpF1iGn](https://github.com/vercel/community/assets/3776997/b8e6b1bf-90d1-414e-b210-71d6e2f9746c)
![chrome_qPLeNFJRAo](https://github.com/vercel/community/assets/3776997/0827db99-6cbd-4398-b5a4-eb223190878d)
![chrome_rKhGjgecJQ](https://github.com/vercel/community/assets/3776997/8cc738ed-c4f5-48e8-8eb2-5f4b8ac887e5)

**Comment by amyegan** [comment, marked as answer]
Hey @carstenhag. It looks like your account may recently have been updated to a new structure. You can read the original announcement at [vercel.com/changelog/2024-01-account-changes](https://vercel.com/changelog/2024-01-account-changes)

I've put together a list of some common points of confusion and solutions. That's posted in https://github.com/vercel/community/discussions/6298

**Reply by carstenhag** [reply]
Thanks, that was easier to resolve than expected :)
